SmartCity/README.md

```diff
@@ -0,0 +1,41 @@
+# MatrixOne SmartCity Showcase
+
+This project takes MatrixOne open source distributed HSTAP database and open source streaming distributed storage Pravega as the core to build an end-to-end smart city digital system, including video acquisition, transmission, model reasoning, data cleaning, storage, analysis, visualization and other functional modules. On the basis of system implementation, BLOB data types are implemented for MatrixOne to enrich the data types supported by MatrixOne.
```

Comment:
remove "On the basis of system implementation, BLOB data types are implemented for MatrixOne to enrich the data types supported by MatrixOne."